lib/tasks/doctor.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+var defaultDoctor = require('./doctor');
```

Comment:
This line confused me first reading it. I thought you were somehow requiring the same file we're in now. Anyone like the idea of throwing `/index` on the end?

It's not a big deal, just wanted to throw the idea out there for some feedback